src/cart/cart.service.mongo.ts

```diff
@@ -0,0 +1,84 @@
+import { BaseError } from "../exceptions/BaseError";
```

Comment:
Please, ignore the files with mongo in their names. They are from the previous module. I just renamed them for keeping into the repository.